README.md

```diff
@@ -0,0 +1,85 @@
+# Zerobase API Server
+
+This repository contains the back end for the Zerobase smart tracing platform. Refer to the [smart-tracing repo](https://github.com/zerobase-io/smart-tracing) for the front end. We are using Kotlin and Neo4j with cloud storage on Heroku. We are using Dropwizard for the REST framework. Kotlin requires Java and Maven.
+
+# TODO
+
+*
+
+## Set up for the project
+
+### Kotlin
+
+* The backend is written in Kotlin. Make sure you have a compatible IDE such as [IntelliJ](https://www.jetbrains.com/idea/download/index.html?_ga=2.137859766.761208892.1584829709-1795868819.1584829709#section=mac).
+
+
+### Java 11
+
+* This project requires Java version 11 which can be dowloaded [here](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
+
+### Maven
+
+* Download Maven from [here](https://maven.apache.org/download.cgi)
+* Follow these installation instructions [here](https://maven.apache.org/install.html)
+* Add the full path to your ~/.bash_profile 
+
+
+### Dropwizard
+
+* A library for building RESTful web services [here](https://github.com/dropwizard/dropwizard) 
+* Maven is required to run successfully
+
+### Docker
+
+Docker is used to run Neo4j on this project 
+
+*  Install docker [here](https://www.docker.com/get-started?utm_source=google&utm_medium=cpc&utm_campaign=getstarted&utm_content=sitelink&utm_term=getstarted&utm_budget=growth&gclid=EAIaIQobChMIzsLmsdWu6AIVA4bICh3VWArbEAAYASABEgKP8_D_BwE)
+* Pull the Neo4j image
+
+```sh
+$ docker pull neo4j
+```
+
+### Project
+
+* After cloning the project there are many ways to deploy it locally. The following directions pertain to deployment via the IntelliJ IDE. In whichever environment you choose, ensure you're using Java 11 and set the environment variable `GRAPHENEDB_PASSWORD=zerobase`. Run the server via `server src/main/resources/config.yml`
+
+
+*In IntelliJ
+	* Navigate to File/Project Structure. Update the JDK home path to Java 11.
+	* In the file path src/main/kotlin/models/ open `Main.kt`
+	* In `Main.kt` right click on the run button next to main and click Edit Run Configuration
+	* ![main](https://github.com/alh2202/smart-tracing-api/blob/master/main.png)
```

Comment:
the full url for images shouldn't point to your repo. this should probably be a relative url. also, i don't see the image files added to the repo as part of the PR. same for the one below